crates/revm/src/db/in_memory_db.rs

```diff
@@ -430,7 +431,7 @@ mod tests {
 
     #[test]
     fn test_replace_account_storage() {
-        let account = 42.into();
+        let account = Address::with_last_byte(42);
```

Comment:
Why don't we have something like `Address::from(42)`?
nit for future, fine with change rn.